cfme/tests/services/test_dialog_regex_validation_in_catalog.py

```diff
@@ -25,16 +25,15 @@ def dialog_cat_item(appliance, catalog, request):
             'ele_label': fauxfactory.gen_alphanumeric(15, start="ele_label_"),
             'ele_name': fauxfactory.gen_alphanumeric(15, start="ele_name_"),
             'ele_desc': fauxfactory.gen_alphanumeric(15, start="ele_desc_"),
-            'choose_type': "Text Box"
+            "choose_type": hasattr(request, "param") and request.param.get
```

Comment:
I don't think this is behaving quite in the way you intended.

the value for the dict field will be set to the boolean result of `hasattr() and request.param.get()` - not the string value of `request.param.get()`